lib/CleantalkSP/SpbctWP/DB/SQLSchema.php

```diff
@@ -103,6 +103,46 @@ class SQLSchema extends \CleantalkSP\Common\DB\SQLSchema
             ),
         ),
 
+        'scan_results_log' => array(
```

Comment:
We should stand the common code style of the document. We'll reformat it to PSR-12 in one single commit later.
Please, write the column description in one single line like other columns descriptions.

Reply:
Different ways are used in this code